src/Microsoft.AspNet.Mvc.Core/Filters/IFormatFilter.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    ///  A filter which produces a desired content type for the current request. 
+    /// </summary>
+    /// <remarks>A FormatFilter decides what content type to use if a format value is present in the URL. 
+    /// </remarks>
+    public interface IFormatFilter : IFilter
+    {
+        /// <summary>
+        /// Returns <c>true</c> if the filter will produce a content type for the current request, otherwise 
+        /// <c>false</c>.
+        /// </summary>
+        /// <param name="context">The <see cref="FilterContext"/></param>
+        /// <returns>If the filter will execute</returns>
```

Comment:
Don't just say `If the filter will execute`

There are generally two styles of documenting a boolean return value of a method:
1). `<True> if ...blahblah.., otherwise <false>.`
2). `A value indicating whether or not ...blahblah....`

It's boilerplate and it gets old, it's also pretty brainless and quick once you get used to the common phrasings used in doc strings.